src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -35,12 +39,12 @@ async function assume_role(req) {
                 Credentials: {
                     AccessKeyId: access_keys.access_key.unwrap(),
                     SecretAccessKey: access_keys.secret_key.unwrap(),
-                    Expiration: expiry,
+                    Expiration: expiration_time_sec,
```

Comment:
This [API](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html#API_AssumeRole_Examples)  should return this element as ISO date string, like - `<Expiration>2019-11-09T13:34:41Z</Expiration>`, so we can't just return it as a number (regardless if the units is sec or ms).

Use the same function that S3 APIs uses - `s3_utils.format_s3_xml_date(time)` and send it the expiration time (in ms which is the standard units for js dates).

```suggestion
                    Expiration: s3_utils.format_s3_xml_date(expiration_time),
```